pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -478,35 +478,45 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 		volumeAttacher, _ = attachableVolumePlugin.NewAttacher()
 	}
 
+	// get deviceMounter, if possible
+	deviceMountableVolumePlugin, _ := og.volumePluginMgr.FindDeviceMountablePluginBySpec(volumeToMount.VolumeSpec)
```

Comment:
Can you add unit tests for these new plugin scenarios?

Reply:
I am not sure i understand the `scenarios`. Just added some UTs for Concurrent Mount operations for devicemountable plugins in the third commit. PTAL, thanks

Reply:
We haven't implemented any DeviceMountable-only plugins for now, this PR is just interface refactor. We can add these UTs in my another PR: https://github.com/kubernetes/kubernetes/pull/63011
WDYT ?